packages/ui-components/src/Input/CommonInput.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import classNames from "classnames/bind";
+import "./input.module.scss";
+
+export interface CommonInputProps {
+  id?: string;
+  // this should be the element containing the input
+  fieldInput?: JSX.Element;
+  className?: string;
+  disabled?: boolean;
+  help?: string | JSX.Element;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  error?: any;
```

Comment:
This is set to any because error in an Input is assigned a value of` meta.touched && meta.error`, and `meta.error` is assigned a type of `any`. I'll add that comment in the code.